docs/examples/parsel-crawler.mdx

```diff
@@ -0,0 +1,50 @@
+---
+id: parsel-crawler
+title: Parsel crawler
+---
+
+This example shows how to use `ParselCrawler` to crawl a website or a list of URLs. Each URL is loaded using a plain HTTP request and the response is parsed using [Parsel](https://pypi.org/project/parsel/) library which supports CSS and XPath selectors for HTML responses and JMESPath for JSON responses. We can extract data from all kinds of complex HTML structures using XPath. In this example, we will use Parsel to crawl github.com and extract page title, URL and emails found in the webpage. The default handler will scrape data from the current webpage and enqueue all the links found in the webpage for continuous scraping.
+
+
+```python
+import asyncio
+
+from crawlee.parsel_crawler import ParselCrawler, ParselCrawlingContext
+
+
+async def main() -> None:
+    crawler = ParselCrawler(
+        # Limit the crawl to max requests. Remove or increase it for crawling all links.
+        max_requests_per_crawl=10,
+    )
+
+    # Regex for identifying email addresses on a webpage.
+    EMAIL_REGEX = r'[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}'
```

Comment:
I'm conflicted about parsing e-mail addresses with regexp like this. The presented use case is cool, but there is almost certainly some mistake here.